docs/user-guide/src/SUMMARY.md

```diff
@@ -27,6 +27,7 @@
       - [External Inspection](bmo/external_inspection.md)
       - [Live ISO](bmo/live-iso.md)
       - [Status Annotation](bmo/status_annotation.md)
+      - [Advanced Instance Customization](bmo/advanced_instance_customization.md)
```

Comment:
Please keep the list sorted and update a similar list in features.md

Reply:
Thanks, done!